<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoiseSpec</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="MainWindowSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>7</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="ChartLayout"/>
      </widget>
      <widget class="QTabWidget" name="SettingsTabWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="PlotSettingsTab">
        <attribute name="title">
         <string>Plot Settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QSplitter" name="PlotSettingsSplitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="PlotDataLayout">
             <item>
              <widget class="QLabel" name="PlotDayaLabel">
               <property name="text">
                <string>Plot Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="DPATable">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="PlotDataButtonsLayout">
               <item>
                <widget class="QPushButton" name="deleteButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Delete</string>
                 </property>
                 <property name="shortcut">
                  <string>Backspace</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="AddButton">
                 <property name="text">
                  <string>Add Signal</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="PlotAxisGroupBox">
            <property name="title">
             <string>Axis </string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QFrame" name="PlotTypeFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QRadioButton" name="RawDataRadio">
                  <property name="text">
                   <string>Raw Data</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="FFTRadio">
                  <property name="text">
                   <string>FFT</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QFormLayout" name="xAxisBoundsLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>x min</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="xMinBox"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>x max</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="xMaxBox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="yAxisBoundsLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>y min</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="yMinBox"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>y max</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="yMaxBox"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QCheckBox" name="LogXCheckBox">
                 <property name="text">
                  <string>Log x axis</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="LogYCheckBox">
                 <property name="text">
                  <string>Log y axis</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuTiming">
     <property name="title">
      <string>Timing</string>
     </property>
     <addaction name="actionManual"/>
     <addaction name="actionAutomatic"/>
    </widget>
    <addaction name="menuTiming"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAutomatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>DPATable</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>RawDataRadio</tabstop>
  <tabstop>FFTRadio</tabstop>
  <tabstop>xMinBox</tabstop>
  <tabstop>xMaxBox</tabstop>
  <tabstop>LogXCheckBox</tabstop>
  <tabstop>yMinBox</tabstop>
  <tabstop>yMaxBox</tabstop>
  <tabstop>LogYCheckBox</tabstop>
  <tabstop>SettingsTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
